packages/web-app-media-viewer/src/App.vue

```diff
@@ -1,100 +1,106 @@
 <template>
-  <main id="mediaviewer" class="uk-position-relative">
-    <div class="uk-text-center oc-p-s">
-      <transition
-        name="custom-classes-transition"
-        :enter-active-class="activeClass.enter"
-        :leave-active-class="activeClass.leave"
+  <main
+    id="mediaviewer"
+    tabindex="-1"
+    @keydown.left="prev"
+    @keydown.right="next"
+    @keydown.esc="closeApp"
+  >
+    <h1 class="oc-invisible-sr" v-text="pageTitle" />
+    <transition
+      name="custom-classes-transition"
+      :enter-active-class="activeClass.enter"
+      :leave-active-class="activeClass.leave"
+    >
+      <div
+        v-show="!loading && activeMediaFileCached"
+        class="
+          uk-width-1-1 uk-flex uk-flex-center uk-flex-middle
+          oc-p-s
+          uk-box-shadow-medium
+          media-viewer-player
+        "
       >
-        <div v-show="!loading && activeMediaFileCached">
-          <video
-            v-if="medium.isVideo"
-            :key="`media-video-${medium.id}`"
-            class="uk-box-shadow-medium media-viewer-player"
-            controls
-            preload
-          >
-            <source :src="medium.url" :type="`video/${medium.ext}`" />
-          </video>
-          <img
-            v-else
-            :key="`media-image-${medium.id}`"
-            :src="medium.url"
-            :alt="medium.name"
-            :data-id="medium.id"
-            class="uk-box-shadow-medium media-viewer-player"
-          />
-        </div>
-      </transition>
-    </div>
+        <video v-if="medium.isVideo" :key="`media-video-${medium.id}`" controls preload>
+          <source :src="medium.url" :type="`video/${medium.ext}`" />
+        </video>
+        <img
+          v-else
+          :key="`media-image-${medium.id}`"
+          :src="medium.url"
+          :alt="medium.name"
+          :data-id="medium.id"
+        />
+      </div>
+    </transition>
     <div v-if="loading" class="uk-position-center">
-      <oc-spinner :aria-label="$gettext('Loading media')" size="xlarge" />
+      <oc-spinner :aria-label="$gettext('Loading media file')" size="xlarge" />
     </div>
     <oc-icon
       v-if="failed"
       name="review"
       variation="danger"
       size="xlarge"
       class="uk-position-center uk-z-index"
-      :accessible-label="$gettext('Failed to load media')"
+      :accessible-label="$gettext('Failed to load media file')"
     />
 
     <div class="uk-position-medium uk-position-bottom-center media-viewer-details">
+      <p class="oc-text-lead uk-text-center oc-text-truncate oc-p-s media-viewer-file-name">
+        <span class="oc-invisible-sr" v-text="currentMediaLabel" />
```

Comment:
Hmmm... the content of the `h1` and of this label are so similar that I'd take back my previous opinion and say that we don't need this string + make the next line `aria-hidden="true"`.